plugin/trino-spooling-filesystem/src/main/java/io/trino/spooling/filesystem/FileSystemSpoolingManager.java

```diff
@@ -127,16 +131,16 @@ public SpooledLocation location(SpooledSegmentHandle handle)
     {
         // Identifier layout:
         //
-        // ttl: long
-        // nameLength: int
-        // name: byte[]
+        // ulid: byte[128]
```

Comment:
I think you mean `byte[16]`

Reply:
Correct. I was thinking about bits obviously :)